content/guides/faq.md

```diff
@@ -21,7 +21,7 @@ Other smaller differences include: a different viewpoint about how schemas shoul
 
 ## Why create Lexicon instead of using JSON-LD or RDF?
 
-ATP exchanges data and RPC commands across organizations. For the data and RPC to be useful, the software needs to correctly handle schemas created by separate teams. This is the purpose of [Lexicon](/guides/lexicon).
+Atproto exchanges data and RPC commands across organizations. For the data and RPC to be useful, the software needs to correctly handle schemas created by separate teams. This is the purpose of [Lexicon](/guides/lexicon).
 
 We want engineers to feel comfortable using and creating new schemas, and we want developers to enjoy the DX of the system. Lexicon helps us produce strongly typed APIs which are extremely familiar to developers and which provides a variety of runtime correctness checks (which are vital in distributed systems).
```

Comment:
expanding DX to "developer experience" might be clearer

Reply:
thanks for the suggestion, just pushed a clarification for this!